### PR TITLE
Add online activity recognition demo

### PR DESCRIPTION
**Pull request by LukasHedegaard**
This pull-request adds a demo for online-activity recognition via a local webcam and webserver.

### CODE REVIEW

File: projects/perception/activity_recognition/demos/online_recognition/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+<head>
+  <title>OpenDR activity recognition demo</title>
+</head>
+
+<body style="background-color:black; text-align:center">
+  <!-- <h1>OpenDR activity recognition demo</h1> -->
+  <img src="{{ url_for('video_feed') }}" alt="OpenDR activity recognition demo"
+    style="height:98vh; max-width:98%; object-fit: contain;">
+  <!-- <a href="https://github.com/LukasHedegaard/opendr-stream-video-browser">GitHub</a> -->
```
Comment (passalis) [inline projects/perception/activity_recognition/demos/online_recognition/templates/index.html:11]: URLs should point to official OpenDR repo.

**Review by passalis — COMMENTED** [review]
Thanks for this demo. There are a few comments to be addressed.

File: projects/perception/activity_recognition/demos/online_recognition/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
[the remark this answers is not in the document]
Comment (LukasHedegaard) [reply projects/perception/activity_recognition/demos/online_recognition/LICENSE]: You are right, it's redundant here. Deleted in 
21d17b7

File: projects/perception/activity_recognition/demos/online_recognition/templates/index.html (hunk 1 above)
Comment (LukasHedegaard) [reply projects/perception/activity_recognition/demos/online_recognition/templates/index.html]: You are right. The link was a commented out remnant of prior code. It was deleted in af12c3e

**Review by passalis — APPROVED** [review]
Thanks!

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you

**Pull request merged**